static/app/views/onboarding/onboardingController.tsx

```diff
@@ -12,7 +13,12 @@ type Props = Omit<React.ComponentPropsWithoutRef<typeof Onboarding>, 'projects'>
 };
 
 function OnboardingController({experimentAssignment, ...rest}: Props) {
-  // TODO: call logExperiment
+  React.useEffect(() => {
```

Comment:
@Neo-Zhixing Can we remove this until after we've tested the new flow out on prod? This was supposed to be handled in a future PR: https://getsentry.atlassian.net/browse/GRW-473